### PR TITLE
Pink Security

### PR DESCRIPTION
**Pull request by Youtubeboy139**
## About The Pull Request

ATTENTION:
In celebration of government sanctioned Pride Month as well as Breast Cancer Awareness, NanoTrasen has temporarily swapped out the uniforms of the Security Department with pink variations. Reminder, laughing at security officers wearing this new uniform is ~~encouraged~~ frowned upon and will be met with ~~praise~~ disciplinary actions in the form of ~~increased~~ docked pay.

Screenshot_2023-06-09_140823
## Why It's Good For The Game
https://www.youtube.com/watch?v=ZyhrYis509A

## Changelog
:cl:
add: Added security specific gloves
image: Changed all standard security uniforms pink
/:cl:

### CODE REVIEW

**Comment by EricZilla** [comment]
bimbo sec

**Comment by Fikou** [comment]
image
i do gulag all assistants, how did you guess?

**Comment by Cheshify** [comment]
These go hard, keep em year round

**Comment by DrDiasyl** [comment]
If this won't be test merged for at least 3 days, I will protest.

File: code/modules/clothing/gloves/special.dm [hunk 1]
```diff
@@ -153,3 +153,9 @@
 	siemens_coefficient = 0.3
 	clothing_traits = list(TRAIT_QUICKER_CARRY, TRAIT_CHUNKYFINGERS)
 	clothing_flags = THICKMATERIAL
+
+/obj/item/clothing/gloves/security
+	name = "security gloves"
+	desc = "A pair of gloves, they don't look special in any way."
+	icon_state = "sec_gloves"
+	greyscale_colors = null
```
Comment (necromanceranne) [inline code/modules/clothing/gloves/special.dm:161]:
```suggestion
	greyscale_colors = null

```

**Comment by DrDiasyl** [comment]
I LOVE ORANGES.
ORANGES BEST HEADCODER.
ORANGES IS ALWAYS GETTING W'S.
NOBODY CAN STOP ORANGES.
ORANGES FOR HEADMIN, ORANGES FOR HOST.

**Comment by Iamgoofball** [comment]
did you remember to recolor security pink too

**Review by Wallemations — APPROVED** [review]
Ideal sec.

**Comment by Loafin34** [comment]
recolor HoS as well.

**Comment by Youtubeboy139** [comment]
> recolor HoS as well.

I would but the colors available are pure white or purple like the Warden and I wanted the two outfits to have distinction at a glance

**Comment by CCC23** [comment]
FEMBOY SEC?

**Comment by Muffindrake** [comment]
I love you for implementing this, I brought pink security up the other day in the discord.

**Comment by TheVekter** [comment]
Can we make this a rare trait after the funny haha period is over

**Comment by DrDiasyl** [comment]
> Can we make this a rare station trait after the funny haha period is over



Mothblocks really disliked this idea...

**Comment by lessthnthree** [comment]
> did you remember to recolor security's department pink too

Funny that I just made this very easy to do! https://github.com/tgstation/tgstation/pull/75665

**Comment by Thunder12345** [comment]
You didn't do helmets

**Comment by Mothblocks** [comment]
@DrDiasyl no the pr just made it seem like it didnt want to be merged so i gave it the fancy label i give these kinds of prs

**Comment by LemonInTheDark** [comment]
@Youtubeboy139 I can't tm it if it's conflicting

**Comment by YuliceTheDruid** [comment]
MERGED?

**Comment by Youtubeboy139** [comment]
Ayo, what? Why the fuck was this merged?

**Comment by DrDiasyl** [comment]
> > IT GOT MERGED? HELL YEAH.
> > 
> > Anyone who is upset - cope.
> 
> I'm upset and I'm the one who made the PR, this thing was never meant to be merged

That's unfortunate turn of events, I'm sure everyone will like this.
~~and anyway this was merged just to buf people, and it will be reverted sometime later~~

**Comment by at996** [comment]
archive this for april fools, you won't.

**Comment by Youtubeboy139** [comment]
> archive this for april fools, you won't.

Oh don't worry I'll do something more cursed for April fools possibly

**Pull request merged**